service/config/index.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// Configuration : Struct to contain all configuration
+type Configuration struct {
+	KeywordPriority []string `json:"keywordPriority"`
+	UserAgents []string `json:"userAgents"`
+}
+
+var config Configuration
+
+// Init reads and stores data from config.json
+func Init() {
```

Comment:
@irshadjsr21 you can change the function name to `inti()`, in golang all the `init` functions in any of the files gets executed as soon as the server is started.

Reply:
Didn't knew that. Thanks for the information. I'll change it.